### ISSUE TITLE
[Github Actions] `publish-to-testpypi` fails on forks 

### ISSUE THREAD

**Issue by Seb-sti1** [ricklupton/rmscene#43]
When I push to my fork, it tries to run `publish-to-testpypi`. Of course, it fails but then this marks all push as failed (e.g. https://github.com/Seb-sti1/rmscene/branches).

Would you mind if I added a `if: github.repository == 'ricklupton/rmscene'` on the `publish-to-testpypi` config?

**Comment by ricklupton** [comment]
Makes sense, thanks. I based this on the PyPI template but it is excessive. 

It would make sense to only run on main branch, too, if you can easily do that at the same time.
